recipes/estimagic/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "estimagic" %}
+{% set version = "0.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/estimagic-{{ version }}.tar.gz
+  sha256: 1c53c49a6e8ea14dae72fd5c04243011ff4de44edc88e21012f74a1904687725
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - estimagic=estimagic.cli:cli
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - bokeh >=1.3
+    - click
+    - cloudpickle
+    - dill
+    - fuzzywuzzy
+    - joblib
+    - numpy >=1.16
+    - pandas >=1.0
+    - python >=3.8
+    - scipy
+    - seaborn
+    - sqlalchemy >=1.3
+    - chaospy
+
+test:
+  imports:
+    - estimagic
+    - estimagic
```

Remove duplicated line:
```suggestion
    - estimagic
```